### ISSUE TITLE
flickr downloading broken :  API change  -  must use ssl?

### ISSUE THREAD

**Issue by Corin-EU** [thesamet/webilder#49]
webilder_downloader 

Downloading list of photos (may take some time) 
url =  http://api.flickr.com/services/rest/?sort=interestingness-desc&extras=original_format&tag_mode=all&tags=autumn&per_page=20&api_key=045379bc5368502f749af23d95a17c83&method=flickr.photos.search
Traceback (most recent call last):
  File "/usr/local/bin/webilder_downloader", line 9, in <module>
    load_entry_point('Webilder==0.7.2', 'console_scripts', 'webilder_downloader')()
  File "/usr/local/lib/python2.7/dist-packages/webilder/downloader.py", line 195, in main
    download_all(notify)
  File "/usr/local/lib/python2.7/dist-packages/webilder/downloader.py", line 150, in download_all
    photos = get_full_download_list(config)
  File "/usr/local/lib/python2.7/dist-packages/webilder/downloader.py", line 26, in get_full_download_list
    photos = module.get_download_list(config_obj)
  File "/usr/local/lib/python2.7/dist-packages/webilder/flickr/**init**.py", line 51, in get_download_list
    photos = flickr.photos_search(*_params_dict) # pylint: disable=W0142
  File "/usr/local/lib/python2.7/dist-packages/webilder/flickr/flickrapi.py", line 44, in photos_search
    photos = self.call('flickr.photos.search', *_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/webilder/flickr/flickrapi.py", line 32, in call
    resp = urllib2.urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(_args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(_args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 403: Forbidden

Trying  the above URL

 http://api.flickr.com/services/rest/?sort=interestingness-desc&extras=original_format&tag_mode=all&tags=autumn&per_page=20&api_key=045379bc5368502f749af23d95a17c83&method=flickr.photos.search

in a web browser results in 

```
<rsp stat="fail"><err code="95" msg="SSL is required"/></rsp>
```

webilder no longer downloads from flickr and  it appears that it needs to be changed to use https (ssl)

Of course the webshots downloading has been broken for years but now that webshots is under new management and offers a free daily photograph in JPG format, surely that could be easily fixed.

**Comment by thesamet** [comment]
The Flickr API has been fixed in 0.7.3 (released last week).

Happy to take PR that fix Webshots support.

**Comment by Corin-EU** [comment]
OOPS -  Sorry, I failed to notice the brand new version  0.7.3 just released on 2014-07-14.

I have just installed it and very happy to report that downloading is working just fine with the new https URLS

webilder_downloader 
Downloading list of photos (may take some time) 
url =  https://api.flickr.com/services/rest/?sort=interestingness-desc&extras=original_format&tag_mode=all&tags=autumn&per_page=20&api_key=045379bc5368502f749af23d95a17c83&method=flickr.photos.search
...

Thank you for continuing to maintain webilder!
